.circleci/nuke_config.yml

```diff
@@ -272,3 +272,9 @@ MSKCluster:
       - "^msk-cluster-[a-zA-Z0-9]{6}$"
       - "^msk-cluster-[a-zA-Z0-9]{6}-.*"
       - "^msk-tiered-storage-test.*"
+  
+InternetGateway:
+  exclude:
+    names_regex:
+      # Exclude the IGW for default VPCs in phxdevops
+      - "gw-new-default-igw"
```

Comment:
Sorry to be a pain, but could you make even more context? This says _what_ it does, but not _why_?

Something like, "We use the Default VPC in many of our tests, and without an IGW, ..."